system-tests/README.md

```diff
@@ -1,62 +1,106 @@
 ## System tests
 
-The test uses the key vault secret to connect to the storage accounts and copy a file from provider to consumer storage account.
+The test copy a file from provider to consumer blob storage account.
 
 ### Building MVD project
 
-One of the MVD dependencies is the Registration Service REST client library. Registration Service is not published to any central artifactory yet so in local 
-development we have to use locally published dependencies.
+MVD dependencies are Eclipse DataSpaceConnector(EDC) and Registration Service. Both of these dependencies are not published to any central artifactory yet so in local
+development we have to use locally published dependencies, once this is done MVD can be build using
 
-#### Publish Registration Service to local Maven
+```bash
+./gradlew build
+```
+
+#### Publish EDC and Registration Service to local Maven
+
+Checkout [Eclipse DataSpaceConnector repository](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector).
 
-Checkout [Registration Service repository](https://github.com/agera-edc/RegistrationService). 
+Publish EDC libraries to local Maven artifactory by executing gradle command `./gradlew publishToMavenLocal -Pskip.signing` from EDC root
+folder.
 
-Publish Registration Service libraries to local Maven artifactory by executing gradle command `./gradlew publishToMavenLocal` from Registration Service root 
-folder. 
+Checkout [Registration Service repository](https://github.com/eclipse-dataspaceconnector/RegistrationService).
+
+Publish Registration Service libraries to local Maven artifactory by executing gradle command `./gradlew publishToMavenLocal` from Registration Service root
+folder.
 
 ### Running test locally
 
-Deploy MVD using the GitHub `Deploy` pipeline. We will run EDC instances locally, connected to the storage accounts and key vaults deployed on Azure.
+MVD System tests can be executed locally against a local MVD instance. MVD runs three EDC Connectors and one Registration Service.
 
-From the build result, download the artifact named `testing-configuration` and extract the file `.env` into the `system-tests` directory (note that the file could be hidden in your file explorer due to its prefix).
+First please make sure that you are able to build MVD locally as described in [Building MVD project](#building-mvd-project) section.
 
-In the file, add the application client secret value under the `APP_CLIENT_SECRET` key. It is used to access Key Vault.
+- We need to build EDC Connector launcher and Registration Service launcher.
+- Go to EDC root folder. And execute
```

Comment:
Going to EDC root folder didn't work with error `Task 'shadowJar' not found in project ':launchers'.`.

Should we go to MVD root folder?

Reply:
Yes, updated. Thanks.

Good catch.